source/backend/api/Services/TakeService.cs

```diff
@@ -92,7 +100,20 @@ public IEnumerable<PimsTake> UpdateAcquisitionPropertyTakes(long acquisitionFile
                 }
             }
 
+            // Update takes
             _takeRepository.UpdateAcquisitionPropertyTakes(acquisitionFilePropertyId, takes);
+
+            // Evaluate if the property needs to be updated
+            var currentProperty = _acqFileRepository.GetProperty(acquisitionFilePropertyId);
+            var currentTakes = _takeRepository.GetAllByPropertyId(currentProperty.PropertyId);
+
+            var completedTakes = currentTakes.Where(t => t.TakeStatusTypeCode == "COMPLETE");
```

Comment:
This also needs to check that the completed take has not expired.

Reply:
Just checking with Julian ATM, could be a miss in the requirements.

Reply:
@devinleighsmith got a response from julian: 
M: Do you know if the expiry date on the land act tenure should be taken into account for Ownership calculation?
J: Answer is No  
I have confirmed with business do NOT take into account expiry. 
 Furthermore, I am creating a ticket to hide expiry date field for (a) transfer of admin and control and (b) crown grant.